### ISSUE TITLE
🐛 json schema for `noLabelWithoutControl` incorrectly requires `inputComponents` `labelAttributes` `labelComponents`

### ISSUE THREAD

**Issue by anatawa12** [biomejs/biome#4125]
### Environment information

```block
CLI:
  Version:                      1.9.2
  Color support:                true

Platform:
  CPU Architecture:             aarch64
  OS:                           macos

Environment:
  BIOME_LOG_PATH:               unset
  BIOME_LOG_PREFIX_NAME:        unset
  BIOME_CONFIG_PATH:            unset
  NO_COLOR:                     unset
  TERM:                         "xterm-256color"
  JS_RUNTIME_VERSION:           "v22.6.0"
  JS_RUNTIME_NAME:              "node"
  NODE_PACKAGE_MANAGER:         "npm/10.8.2"

Biome Configuration:
  Status:                       Loaded successfully
  Formatter disabled:           false
  Linter disabled:              false
  Organize imports disabled:    false
  VCS disabled:                 true

Workspace:
  Open Documents:               0
```

### What happened?

1. We configured `inputComponents` for our components
2. IDE says `labelAttributes` and `labelComponents` are required attributes, but biome cli correctly parses the configuration JSON so I think this is false-positive

### Expected result

I think `labelAttributes` and `labelComponents` should not be marked as required in json schema, if biome cli accepts json without them

### Code of Conduct

- [X] I agree to follow Biome's Code of Conduct

**Comment by anatawa12** [comment]
This is similar to #3763 and there might be several similar. Should I create issue for each time I found or send new ones as comment for some existing issue?

**Comment by ematipico** [comment]
> This is similar to #3763 and there might be several similar other places. Should I create issue for each time I found or send new ones as comment for some existing issue?

Yes please. Every option is different
